docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/field/field--node--field-intro-text.html.twig

```diff
@@ -1,5 +1,7 @@
-<div class="blog-intro-text">
-  {% for item in items %}
-    {{- item.content -}}
-  {% endfor %}
-</div>
\ No newline at end of file
+<div class="cgdp-intro-text">
+	<div class="usa-prose">
```

Comment:
Can you add the prose variant class for the WYSIWYG too please? 

(I don't know if I like this, but here is an example... ) https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/entrypoints/global/ncids-minimal.scss#L59